src/core/AgendaDeck.ttslua

```diff
@@ -1,132 +1,62 @@
-MIN_VALUE = -99
-MAX_VALUE = 999
-
-function onload(saved_data)
-    light_mode = false
-    val = 0
-
+-- Doom Counter with Print
+-- original by:     -
+-- changed by:      Chr1Z
+-- description:     Clickable counter for doom on the agenda, changing the value prints the new value, reset button added
+information = {
+    version = "1.4",
+    last_updated = "12.11.2022"
+}
+
+function onSave() return JSON.encode(val) end
+
+function onLoad(saved_data)
     if saved_data ~= "" then
-        local loaded_data = JSON.decode(saved_data)
-        light_mode = loaded_data[1]
-        val = loaded_data[2]
-    end
-
-    createAll()
-end
-
-function updateSave()
-    local data_to_save = {light_mode, val}
-    saved_data = JSON.encode(data_to_save)
-    self.script_state = saved_data
-end
-
-function createAll()
-    s_color = {0.5, 0.5, 0.5, 95}
-
-    if light_mode then
-        f_color = {1,1,1,95}
+        val = JSON.decode(saved_data)
     else
-        f_color = {0,0,0,100}
+        val = 0
     end
 
-
-
     self.createButton({
-      label=tostring(val),
-      click_function="add_subtract",
-      function_owner=self,
-      position={0,0.05,0},
-      height=600,
-      width=1000,
-      alignment = 3,
-      scale={x=1.5, y=1.5, z=1.5},
-      font_size=600,
-      font_color=f_color,
-      color={0,0,0,0}
-      })
-
-
-
-
-    if light_mode then
-        lightButtonText = "[ Set dark ]"
-    else
-        lightButtonText = "[ Set light ]"
-    end
-
-end
+        label          = tostring(val),
+        click_function = "add_subtract",
```

Comment:
Nit: Please rename so the style is consistent with other methods (i.e. addSubtract).  I'd probably rename it to addOrSubtract as well